src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -177,9 +202,16 @@ class ActivityFeed extends React.Component<Props, State> {
                     }}
                     className="bcs-activity-feed-items-container"
                 >
-                    {isEmpty ? (
-                        <EmptyState isLoading={isLoading} showCommentMessage={showCommentForm} />
-                    ) : (
+                    {isLoading && (
```

Comment:
We ended up pulling this loader out of `<EmptyState>` because the logic for empty and loaded was so conflated and confusing (and buggy).